tests/unit/commands/test_remote.py

```diff
@@ -412,29 +410,6 @@ def test_run_in_shallow_repo_unsupported(
 ######################
 
 
-def test_no_architecture_all(
```

Comment:
should we have the "opposite" test now? That is, test that 'all' *is* supported?

Reply:
I was on the fence about this because it doesn't test any code or behavior specific to `snapcraft.commands.remote.py` and I already have spread tests for this.

On the other hand, `all` doesn't have a history of good support so I added unit tests.